veracruz-mcu-client/tap-setup.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+#
+# ## Authors
+#
+# The Veracruz Development Team.
+#
+# ## Licensing and copyright notice
+#
+# See the `LICENSE.markdown` file in the Veracruz root directory for
+# information on licensing and copyright.
+#
+
+ip tuntap add zeth mode tap
+ip link set dev zeth up
+ip link set dev zeth address 00:00:5e:00:53:ff
+ip address add 193.0.2.2/24 dev zeth
```

Comment:
What are these IP addresses?  Are they arbitrary?

Reply:
This is arbitrary, we're creating our own NAT inside the Docker, which is required to connect the tap interface that passes raw eth packets to Zephyr.

Since it's behind Docker and client side (isn't shared) I don't think it's worth making it configurable, but let me know if you disagree

Reply:
No, that's fine, I think.